docs/src/main/sphinx/connector/phoenix.rst

```diff
@@ -44,10 +44,7 @@ For HBase 2.x and Phoenix 5.x (5.1.0 or later) use:
 
     connector.name=phoenix5
 
-Configuration properties
-------------------------
-
-The following configuration properties are available:
+The following Phoenix-specific configuration properties are available:
 
```

Comment:
I'd rephrase to remove "are available" it sounds awkward. I'd recommend something like:

>Review the available Phoenix-specific configuration properties: